rmgpy/molecule/atomtype.py

```diff
@@ -235,7 +235,7 @@ def getFeatures(self):
     'O','Os','Od','Oa','Ot',
     'Ne',
     'Si','Sis','Sid','Sidd','Sit','SiO','Sib','Sibf',
-    'S','Ss','Sd','Sa','St',
+    'S','S0s','Sa','S2s','S2sp','S2sn','S2d','S2dc','S4s','S4sc','S4d','S4dc','S4b','S4dd','S4t','S4tc','S6s','S6d','S6dc','S6dd','S6ddc','S6ddd','S6t','S6td','S6tt',
```

Comment:
Definitely. I just added a simple docstring describing the naming convention for these atomTypes.
I'll also push an updated atomType documentation